### DISCUSSION TITLE
sorry!when can we use this project as a golang library?[feature]

### DISCUSSION THREAD

**Discussion by Nyx2022** [projectdiscovery/naabu#205] [General]
**As this title**
I wanna know that when can we use naabu like project subfinder's EnumerateSingleDomain method in our project?maybe i'm too anxious

**Comment by Nyx2022** [comment]
i mean i wanna give a input like []string to naabu and receive a result like []Result,
so can realise this goal?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
